source/extensions/transport_sockets/tls/io_handle_bio.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "envoy/network/io_handle.h"
+
+#include "openssl/bio.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace TransportSockets {
+namespace Tls {
+
+// NOLINTNEXTLINE(readability-identifier-naming)
+const BIO_METHOD* BIO_s_io_handle(void);
```

Comment:
This method seems to only be used in the cc file, should we remove it from the header?

Reply:
Good point. I exposed it for testing but ended up not using it.